extensions/yaml_support/labgraph_monitor/server/lg_monitor_server.py

```diff
@@ -30,26 +21,18 @@
 SAMPLE_RATE = 5
 
 
-def run_server(data: SerializedGraph) -> None:
+def run_server(graph: lg.Graph, data: SerializedGraph) -> None:
     """
     A function that creates a Websocket server graph.
     The server graph streams the lagraph topology to the clients
     """
+    # Setup original graph first
+    graph.setup()
```

Comment:
1. I'm setting up `Demo` graph here to run it simultaneously with `WSSenderNode`